### PR TITLE
Annotate by passing `text-file` to the impl layer

### PR DESCRIPTION
**Pull request by hrshdhgd**
- [x] Added a new param `forward_text_file` to `TextAnnotationCongfiguration`

### CODE REVIEW

File: pyproject.toml [hunk 1]
```diff
@@ -1,6 +1,6 @@
 [tool.poetry]
 name = "oaklib"
-version = "0.0.0"
+version = "0.1.49"
```
Comment (cmungall) [inline pyproject.toml:3]: was this intentional?

File: src/oaklib/cli.py [hunk 2]
```diff
@@ -989,13 +992,17 @@ def term_metadata(terms, reification: bool, output_type: str, output: str):
     "-L",
     help="path to lexical index. This is recreated each time unless --no-recreate is passed",
 )
+@click.option(
+    "--forward-text-file", "-F", help="Implementation level text analysis.", default=False
```
Comment (cmungall) [inline src/oaklib/cli.py:996]: Just `--text-file` is better

Comment (cmungall) [reply src/oaklib/cli.py:996]: actually we have text file, so I don't see the point of this

**Comment by cmungall** [comment]
is this to address #329?

File: pyproject.toml (hunk 1 above)
Comment (hrshdhgd) [reply pyproject.toml:3]: yes for testing purposes ... I intend to roll it back

**Pull request closed without merging**